code/datums/elements/decal.dm

```diff
@@ -4,6 +4,7 @@
 	var/cleanable
 	var/description
 	var/mutable_appearance/pic
+	var/last_dir
```

Comment:
It's a bespoke element now, so this won't properly work, as we can't really store the state of each decal, as that somewhat defeats the point. Is there some other way you could get this info to `shuttle_move_react` without storing it on the element?

Reply:
No. I dont see any way, since pic dont has dir.
I make it transfer pic.